finder-app/finder.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#R eferences: 1. https://www.cyberciti.biz/faq/grep-count-lines-if-a-string-word-matches/
+# 2. https://stackoverflow.com/questions/8509226/using-find-command-in-bash-script
+
+
+# check if there are two arguments 
+if [ $# -ne 2 ]
+then
+	echo "Wrong number of args. Use 2 arguments"
```

Comment:
@PeterBraganza suggestion: Make the error message here and in writer.sh describe the arguments, help the user understand how to execute correctly without requiring them to look at the code.